Mono.Addins/Mono.Addins.csproj

```diff
@@ -49,6 +49,9 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <PackageReference Include="NuGet.Build.Packaging" Version="0.1.276" />
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing">
```

Comment:
I to keep Mono.Addins.dll standalone, without other NuGet dependencies. Either copy the globbing code here or use a custom one.